### DISCUSSION TITLE
StartMenu tile board always takes out and shows three bare tile on the first tile group.

### DISCUSSION THREAD

**Discussion by 5o1** [valinet/ExplorerPatcher#1971] [Q&A]
Everytime I reboot my win11, three consecutive bare tiles appear in the first group of the Start Menu tile panel.

I think they may be old functional tile under win10 like windows calendar, albums, etc. They may not be displayed correctly here. I'm not sure. It is just a guess. 

The only thing I'm sure of is that every time I boot the system I have to eliminate the three empty tiles by dragging the next row of tiles to the previous row with the cursor.

Maybe I can fix this by manually modifying the layout of the startmenu. But I don't know where the layout profile is. Can anyone provide me with help?

----

I tried to apply [this method](https://learn.microsoft.com/en-us/powershell/module/startlayout/export-startlayout?view=windowsserver2022-ps), but I got an empty profile.
`Export-StartLayout -Path "C:\Layouts\Marketing.xml"`

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
